scripts/generate-questions-csv.py

```diff
@@ -12,12 +12,15 @@
 pip install -r requirements.txt
 
 Usage:
-    scripts/generate-questions-csv.py <DM-content-root-dir> <output-directory> --framework=<slug> [--context=<yaml>]
+    scripts/generate-questions-csv.py <DM-content-root-dir> <output-file> [options]
 
+--framework=<slug>
+--question-set=<question-set-directory-name>
+--context=<yaml>
 -h --help    show this message
 
 Example:
-    scripts/generate-questions-csv.py /path/to/dm-frameworks/ ~ --framework=g-cloud-9 --context="lot: SaaS"
+    scripts/generate-questions-csv.py /path/to/dm-frameworks/ ~/mystuff.csv --framework=g-cloud-9 --context="lot: SaaS"
```

Comment:
OK OK, so we do that with `question-set` rather than `context`.

I think the upshot of my comments here is that the docstring for the script should explicitly say that  there are two possible values for `question-set` and that they are `declaration` and `services`.